<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rename</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Consolas&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <widget class="QLineEdit" name="lnEdit_rename">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btn_rename">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #27ae60;
	border-radius: 8px;
	border-style: none;
	color: #fff;
	font-size: 16px;
	font-weight: 600;
	letter-spacing: normal;
	line-height: 1.5;
	outline: none;
	padding: 13px 20px;
	position: relative;
	text-align: center;
	text-decoration: none;
	vertical-align: top;
	white-space: nowrap;
}

QPushButton:pressed{
  background-color: #16cc65;     
}

QPushButton:hover {
	background-color: #0b7538;
}

</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_rename</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_rename</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
